### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by pasha1932**
[DEMO LINK](https://pasha1932.github.io/layout_moyo-header/)
[TEST REPORT LINK](https://pasha1932.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -9,6 +9,22 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <div class="container">
+      <header class="header">
+        <img src="images/logo.png" alt="logo"
+        class="logo">
+        <nav class="nav">
+          <a href="#Apple" class="nav__link is-active">Apple</a>
```
Comment (SanyaBratashchuk) [inline src/index.html:17]:
```suggestion
          <a 
            href="#Apple" 
            class="nav__link is-active"
          >
            Apple
         </a>
```
Fix formatting for similiar cases

**Review by SanyaBratashchuk — CHANGES_REQUESTED** [review]

**Review by pasha1932 — COMMENTED** [review]
Updated src/index.html

File: src/index.html [hunk 2]
```diff
@@ -9,6 +9,32 @@
...
+      <header class="header">
+        <img src="images/logo.png" alt="logo"
+        class="logo">
+        <nav class="nav">
+          <a
+          href="#Apple"
+          class="nav__link is-active"
+          >
+          Apple
+          </a>
```
Comment (SanyaBratashchuk) [inline src/index.html:22]:
```suggestion
        <img 
          src="images/logo.png" 
          alt="logo"
          class="logo"
        >
        <nav class="nav">
          <a href="#Apple" class="nav__link is-active">
            Apple
          </a>
```

**Review by SanyaBratashchuk — CHANGES_REQUESTED** [review]
Fix formatting according to the rules specified in checklist https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md

File: src/index.html [hunk 3]
```diff
@@ -3,12 +3,38 @@
...
+            href="#Laptops & Computers"
+            class="nav__link hover"
+            data-qa="hover"
+          >
+            Laptops & Computers
+          </a>
+          <a href="#Gadgets" class="nav__link">Gadgets</a>
+          <a href="#Tablets" class="nav__link">Tablets</a>
+          <a href="#Photo" class="nav__link">Photo</a>
+          <a href="#Video" class="nav__link">Video</a>
```
Comment (SanyaBratashchuk) [inline src/index.html:35]:
```suggestion
          <a href="#Gadgets" class="nav__link">Gadgets</a>
          <a href="#Tablets" class="nav__link">
            Tablets
          </a>
          <a href="#Photo" class="nav__link">
            Photo
          </a>
          <a href="#Video" class="nav__link">
            Video
          </a>
```

**Review by SanyaBratashchuk — APPROVED** [review]

**Review by SanyaBratashchuk — APPROVED** [review]

**Pull request closed without merging**